pytorch/inference/docker/1.3.1/py3/Dockerfile.eia

```diff
@@ -4,12 +4,12 @@ LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 LABEL dlc_major_version="1"
 
 # Add arguments to achieve the version, python and url
-ARG PYTHON_VERSION=3.6.6
+ARG PYTHON_VERSION=3.6.10
 ARG PYTORCH_VERSION=1.3.1
 ARG TORCHVISION_VERSION=0.4.2
 ARG GRAPHVIZ_VERSION=0.13.2
 ARG MMS_VERSION=1.0.8
```

Comment:
Can be upgraded to https://github.com/awslabs/multi-model-server/releases/tag/v1.1.2

Reply:
Does EI Images support multi-model-server as well as vanilla Images ?